ios/Malwarelytics+RASP.swift

```diff
@@ -95,10 +99,23 @@ extension Malwarelytics {
             return EmulatorInfo(isEmulator: rasp.isEmulator, detectedEmulatorType: "APPLE")
         case "VPN":
             return rasp.isVpnActive
+        case "ON_CALL":
+            return rasp.isOnCall
         default:
             throw ModuleError.invalidParam(paramName: "message")
         }
-    }    
+    }
+    
+    /// Called once, after the service is initialized.
+    /// - Parameter service: Current service instance.
+    func raspOnInit(service: AppProtectionService) {
+        let rasp = service.rasp
+        if rasp.isEmulator {
```

Comment:
Just a note here that this is because we don't expect to change during the lifetime of the app so it's not periodically checked and there is no hook.

Reply:
I was thinking about the same reason, unfortunately, android emits such event, so it's better to be consistent.